modules/virt-installing-fusion-access-operator.adoc

```diff
@@ -29,7 +29,7 @@ This installs the Operator in the `ibm-fusion-access` namespace. If this namespa
 +
 [WARNING]
 ====
-If you install the {FusionSAN} Operator in a namespace other than `ibm-fusion-access`, {FusionSan} does not work.
+You must install the {FusionSAN} Operator in the `ibm-fusion-access` namespace. Installation in any other namespace is unsupported and will not function.
```

Comment:
I think it's enough to leave it as "is not supported". "will not function" is technically [future tense](https://redhat-documentation.github.io/supplementary-style-guide/#tenses-in-release-notes).